src/v/datalake/tests/CMakeLists.txt

```diff
@@ -9,5 +9,18 @@ rp_test(
       v::datalake
     LABELS datalake
     ARGS "-- -c 1"
-  )
-  
\ No newline at end of file
+)
+
+rp_test(
```

Comment:
nit: would encourage gtest for new tests

Reply:
done